lib/core-integration/src/Test/Integration/Scenario/CLI/Miscellaneous.hs

```diff
@@ -32,7 +32,7 @@ spec = do
     it "CLI_VERSION - cardano-wallet shows version" $  do
         (Exit c, Stdout out) <- cardanoWalletCLI @t ["version"]
         let v = L.dropWhileEnd (== '\n') out
-        v `shouldBe` (showFullVersion version gitRevision)
+        v `shouldContain` (showVersion version <> " (git revision: " )
```

Comment:
**[question]**
How o.O ? `showFullVersion` is just `showVersion v <> " (git revision: " <> T.unpack r <> ")"` . The last parenthesis doesn't show up on Windows? Or maybe there's some trailing characters :thinking: ?

Reply:
Not sure why it is like this but the windows testing bundle expects this version -> `2019.11.18 (git revision: 0000000000000000000000000000000000000000)` and the test is failing because of this. (I checked that the binary `cardano-wallet-jormungandr.exe version` shows the correct version, so I just went after @rvl's suggestion for fixing this test https://github.com/input-output-hk/cardano-wallet/issues/1115#issuecomment-561584001).

Reply:
The test exes aren't stamped with the git revision, so it's 000...